Tests/ValetIntegrationTests/ValetIntegrationTests.swift

```diff
@@ -24,6 +24,12 @@ import XCTest
 /// - The Valet Mac Tests target is left without a host app on master. Mac test host app signing requires CI to have the Developer team credentials down in keychain, which we can't easily accomplish.
 /// - Note: In order to test changes locally, set the Valet Mac Tests host to Valet macOS Test Host App, delete all VAL_* keychain items in your keychain via Keychain Access.app, and run Mac tests.
 func testEnvironmentIsSigned() -> Bool {
+    #if targetEnvironment(simulator)
+    if #available(iOS 16, tvOS 16, macOS 13, watchOS 9, *) {
```

Comment:
```suggestion
    #if targetEnvironment(simulator)
    // Xcode 14's simulator environment is signed, but requires entitlements we can't provide in CI.
    if #available(iOS 16, tvOS 16, macOS 13, watchOS 9, *) {
```

Also, can you modify the `/// - Returns: `true` when the test environment is signed.` line above to read `/// - Returns: `true` when the test environment is signed and entitlements are not required by the current environment.`

Reply:
Updated.